README.md

```diff
@@ -1,5 +1,10 @@
+<p align="center">
```

Comment:
I'm going to be honest, I'm not a fan of this change being done in this PR.
We're already working on clarifications and updating our documentation.

Please remove this change to README.md for this PR.